src/server.js

```diff
@@ -236,6 +236,9 @@ export class Server extends EventEmitter {
 
                     this.stats.httpRequestCount++;
                 }
+                if (!handlerOpts.trgParsed) {
```

Comment:
I think the crash can happen earlier above - see:
`const parsed = parseUrl(request.url);`

Reply:
and `if (!parsed.protocol)` below that

Reply:
Oh you are right, weird that it does not crash there, from checking logs it always crashes here, but you are right, I will add the condition there too,